src/components/utils/include/utils/log_message_loop_thread.h

```diff
@@ -52,9 +54,24 @@ typedef struct {
 } LogMessage;
 
 typedef std::queue<LogMessage> LogMessageQueue;
+class LogMessageLoopThreadTemplate
+    : public threads::MessageLoopThread<LogMessageQueue> {
+ public:
+  // Places a message to the therad's queue. Thread-safe.
+  void PostMessage(const Message& message) OVERRIDE {
+    threads::MessageLoopThread<LogMessageQueue>::PostMessage(message);
+
+    // LOG_MESSAGE_QUEUE_SIZE defined in Utils CMakeLists
+    if (GetMessageQueueSize() > LOG_MESSAGE_QUEUE_SIZE) {
+      // TODO (AKutsan): Avoid stopping thread if message queue is full.
+      // Prefere Handle multiple messages from Message loop thread
+      WaitDumpQueue();
```

Comment:
@LuxoftAKutsan, 
Sorry, I've missed that we are waiting, not dropping packets.
In this case we can face with a original issue with slow reading from devices and corrupt all income data from devices (iap and aoa definitely)
So, this fix will mask issue with non-real-time reading from-to external devices!